pkg/controller/logstash/config.go

```diff
@@ -18,6 +18,10 @@ import (
 	"github.com/elastic/cloud-on-k8s/v2/pkg/controller/common/tracing"
 )
 
+const (
+	LogstashConfigFileName = "logstash.yml"
```

Comment:
nit: not from this pr, this constant is in a package named `logstash`:
```suggestion
	ConfigFileName = "logstash.yml"
```